vision/api/VisionTest/VisionTest.cs

```diff
@@ -258,31 +258,31 @@ protected override ConsoleOutput Run(params string[] args)
 
         [Fact]
         public void DetectPdfDocument()
-        {
-            var _pdfFileName = "HodgeConj.pdf";
-            var outputPrefix = "";
-            var localPath = Path.Combine("data", _pdfFileName);
-            var gcsSourceURI = $"gs://{_bucketName}/{_pdfFileName}";
-            ConsoleOutput output;
-
-            string[] cmdArgs = { "ocr", gcsSourceURI, _bucketName, outputPrefix };
-
-            using (var collector = new BucketCollector(_bucketName))
-            {
-                collector.CopyToBucket(localPath, _pdfFileName);
+        {
+            var _pdfFileName = "HodgeConj.pdf";
+            var outputPrefix = "";
+            var localPath = Path.Combine("data", _pdfFileName);
+            var gcsSourceURI = $"gs://{_bucketName}/{_pdfFileName}";
+            ConsoleOutput output;

```

Comment:
Why is this section of the diff here?

_squinting_ my eyes to see a different 👀

Reply:
Just whitespace diff.  Not sure why.  Maybe the source code had \ns instead of \r\ns there.  Append ?w=1 to the github url to hide white space differences.  For example, https://github.com/GoogleCloudPlatform/dotnet-docs-samples/pull/729/files?w=1